src/components/BenefitPlanProjectsSearcher.js

```diff
@@ -239,9 +239,8 @@ function BenefitPlanProjectsSearcher({
 
   const onAdd = () => {
     history.push({
-      pathname: `${benefitPlanId}/${modulesManager.getRef('socialProtection.route.project')}`,
```

Comment:
Is there a reason this config was removed ?

Reply:
because it was unnecessarily verbose and I noticed the route config is not consistently used so I went with readability